src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.ts

```diff
@@ -70,18 +64,47 @@ export class InstructorSearchPageComponent implements OnInit {
         .pipe(finalize(() => this.loadingBarService.hideLoadingBar()))
         .subscribe((resp: [InstructorSearchResult, InstructorSearchResult]) => {
           this.commentTables = resp[0].searchCommentsTables;
-          this.studentTables = resp[1].searchStudentsTables;
+          const searchStudentsTable: SearchStudentsTable[] = resp[1].searchStudentsTables;
```

Comment:
It looks like we should also change the search service also. Let's leave it to another PR. Please raise an issue afterward or continue do it afterward :P.

Reply:
Okay! Will raise and issue and work on it after this PR